pegasus/sites.v3/code.org/public/lms.haml

```diff
@@ -0,0 +1,67 @@
+---
+title: LMS
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/lms-styles.css", rel: "stylesheet"}
+
+%section.top
+  .wrapper
+    .text-wrapper.col-50.pad
+      %h1=hoc_s("lms_page.heading")
+      %p.body-two
+        =hoc_s("lms_page.top_desc")
+      %a.link-button{href: "#"}
+        =hoc_s(:call_to_action_get_started)
+    %figure.col-45{style: "float: right; margin-top: 1.25rem;"}
+      %img.rounded-corners{src: "/images/lms/LMSHeroAnimation.gif", alt: ""}
+    .clear
+
+%section.bg-neutral-light
+  .wrapper
+    .lms-systems
+      %h2
+        =hoc_s("lms_page.top_module.heading")
+      %p.body-two
+        =hoc_s("lms_page.top_module.desc")
+      =view :"lms/lms_provider_cards"
```

Comment:
Thanks for making a `views/lms` subdirectory 🙌

Reply:
Truly following in the footsteps of administrators greatness